agent/handlers/v1/response.go

```diff
@@ -144,6 +153,25 @@ func NewPortBindingsResponse(dockerContainer *apicontainer.DockerContainer, eni
 	return resp
 }
 
+// NewVolumesResponse creates VolumeResponse for a container
+func NewVolumesResponse(dockerContainer *apicontainer.DockerContainer) []VolumeResponse {
+	container := dockerContainer.Container
+	resp := []VolumeResponse{}
```

Comment:
You may want to use `var resp []VolumeResponse` instead, as the marshal of `resp := []VolumeResponse` would be `[]` instead of `null`.